src/attended-event/attended-event.controller.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  Controller,
+  Get,
+  UseGuards,
+  Query,
+  Res,
+  Post,
+  Body,
+  BadRequestException,
+  UnauthorizedException
+} from "@nestjs/common";
+import { GalaxyIntegrationGuard } from "../auth/guards/galaxy-integration.guard";
+import { AttendedEventService } from "./attended-event.service";
+import { AuthGuard } from "@nestjs/passport";
+import { GetUser } from "../user/user-auth.decorator";
+import { User } from "../user/interfaces/user.interface";
+
+@Controller("attended/")
+export class AttendedEventController {
+  constructor(private attendedEventService: AttendedEventService) {}
+
+  @UseGuards(GalaxyIntegrationGuard)
+  @Get("/")
+  async getAttendedEvents(
+    @Query("eventId") eventId,
+    @Query("userAuthId") userAuthId,
+    @Res() res
+  ) {
+    let filter = {};
+    if (eventId) filter = { eventId };
+    if (userAuthId) filter = { userAuthId, ...filter };
+
+    if (!filter)
+      throw new BadRequestException(
+        "Request must container eventId or userAuthId"
+      );
+
+    const attendedObjects = await this.attendedEventService.findAll(filter);
+    return res.send(attendedObjects);
```

Comment:
```suggestion
    return attendedObjects
```

Reply:
Just returning the object causes requests to timeout for some reason. Is it working for you locally?

Reply:
Actually, nevermind, fixed it